apps/site/src/views/Schedule/sections/ClipboardSchedule/ClipboardSchedule.module.scss

```diff
@@ -0,0 +1,31 @@
+@use "bootstrap-utils" as bootstrap;
+@use "zothacks-theme" as theme;
+
+.accordion {
+	--bs-accordion-btn-icon: url("~@/assets/icons/accordion-btn.svg");
+	--bs-accordion-btn-active-icon: url("~@/assets/icons/accordion-btn.svg");
```

Comment:
Note [non-blocking]: we currently have an asymmetrical drop shadow, and the default transition involves rotating the asset making the shadow wobble although not a huge deal.

Reply:
Should we remove the shadow then?